core/stat/base/bucket_leap_array.go

```diff
@@ -30,10 +29,9 @@ func (bla *BucketLeapArray) ResetBucketTo(bw *BucketWrap, startTime uint64) *Buc
 
 // sampleCount is the number of slots
 // intervalInMs is the time length of sliding window
+// sampleCount and intervalInMs must be positive and intervalInMs%sampleCount == 0,
+// the validation must be done before call NewBucketLeapArray
```

Comment:
How about change the param to `bucket span time` and `bucket count` in future？ Is it more easy to use?

Reply:
Is your meaning bucket count and bucketLengthMs?

Reply:
Yes.

Reply:
We could file another PR to enhance here

Reply:
Please new a issue and assign to me.

Reply:
You can submit a issue.